src/controller/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -22,9 +22,35 @@
 namespace chip {
 namespace Controller {
 
+constexpr const char kOperationalCredentialsIssuerKeypairStorage[] = "ExampleOpCredsCAKey";
+
 using namespace Credentials;
 using namespace Crypto;
 
+CHIP_ERROR ExampleOperationalCredentialsIssuer::Initialize(PersistentStorageDelegate & storage)
+{
+    Crypto::P256SerializedKeypair serializedKey;
+    uint16_t keySize = static_cast<uint16_t>(serializedKey.Capacity());
+
+    if (storage.SyncGetKeyValue(kOperationalCredentialsIssuerKeypairStorage, serializedKey, keySize) != CHIP_NO_ERROR)
+    {
+        // Storage doesn't have an existing keypair. Let's create one and add it to the storage.
+        ReturnErrorOnFailure(mIssuer.Initialize());
+        ReturnErrorOnFailure(mIssuer.Serialize(serializedKey));
```

Comment:
This is storing a private key outside of secure storage, where it may leak and be exfiltrated.

Key storage should use different APIs than regular storage, so that crypto-accelerated or trustzone target can rely on their secure storage of the private part of the key pair.

Reply:
@pan-apple ?

Reply:
These are only used for test tools. The intent is not use it for any product release. That's why it's named `ExampleOperationalCredentialsIssuer.cpp`

Reply:
Yes, I understand it's example code. I just see other usages elsewhere of how P-256 keys are used that make assumptions about serialization that are incompatible with iOS/Mac Secure Enclave, Android Keystore, ARM TrustZone in Cortex-M33, etc 

I think it's OK in the short term, but it absolutely violates some of the Threat Model security requirements which are explicit in the spec, so I'm pointing it out early to make sure it doesn't "spread" too much.

Reply:
Agree with @tcarmelveilleux , I'm curious if we can either put this in a place that it's clear it's an example (directory), or make it only optionally compile in somehow?

Reply:
I added a warning and a file note to alert the users of the API about the storage of encryption keys.